quickwit/quickwit-common/src/metrics.rs

```diff
@@ -19,7 +19,7 @@
 
 use std::collections::HashMap;
 
-use once_cell::sync::Lazy;
+use once_cell::sync::{Lazy, OnceCell};
```

Comment:
You could have kept using `Lazy`, but if you prefer the `OnceCell` API, there's an equivalent in the std now since 1.70: [OnceLock](https://doc.rust-lang.org/std/sync/struct.OnceLock.html)